spec/spec_helper.rb

```diff
@@ -85,7 +62,6 @@ def image_path(path, options = {})
 
   config.before(:each) do |example|
     EmsRefresh.try(:debug_failures=, true)
-    ApplicationController.handle_exceptions = false if %w(controller requests).include?(example.metadata[:type])
```

Comment:
This one I did test against the API, and it didn't seem to matter if it was present or not.